CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
```

Comment:
Is the idea still to eventually move this into `@web`? If so, we already have a `CONTRIBUTING.md` in the monorepo

Reply:
yes, I'm OK to remove it for now to not do double work
@michellangeveld are you OK with that too? you initially added this